<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutomateExcel</class>
 <widget class="QDialog" name="AutomateExcel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Adjust File Screen</string>
  </property>
  <widget class="QPushButton" name="btnSelectFiles">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Files</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtConsole">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>260</y>
     <width>481</width>
     <height>271</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Dubai Medium</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Dubai Medium'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTaarcomImage">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>201</width>
     <height>201</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>TAARCOM.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTxtExcelOperations">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>340</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Excel Operations</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTxtCurrentFile">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; text-decoration: underline;&quot;&gt;Current Files:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSelectedFiles">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>331</width>
     <height>211</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;No Files Selected&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClearConsole">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>540</y>
     <width>131</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear Console</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClean">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Clean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCompile">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Compile</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTxtExcelOperations_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>390</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1.</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblTxtExcelOperations_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>440</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDeselectFiles">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>clear-file.png</normaloff>clear-file.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
  </widget>
  <widget class="QComboBox" name="comboCompany">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <item>
    <property name="text">
     <string>Company</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DGK</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MOU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ABR</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btnSplit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>440</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Split</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblTxtExcelOperations_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>490</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
